### PR TITLE
CE-441 Initial update

### PR DESCRIPTION
**Pull request by Convale**
Here's the initial update to update some of the basic stylings and also include ticket status. Most of the functions and HTML were redone. Changed references to Vouch vs prior owner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
+  - Install Zendesk CLI: `npm install @zendesk/zcli -g`
+
+### Setup
+
+- Set up your credentials to Zendesk: `zcli login -i`
+
+### Run
+
+- Start the server: `zcli apps:server`
+- Open zendesk sandbox and add `?zcli_apps=true` to the end of your Zendesk URL
```
Comment (tlhinman) [inline README.md:15]: Capitalize "Zendesk"

File: manifest.json [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+        "url": "assets/iframe.html",
+        "flexible": true
+      }
     }
   },
-  "version": "1.0.6",
+  "version": "1.1.0",
   "frameworkVersion": "2.0"
-}
+}
```
Comment (tlhinman) [inline manifest.json:20]: Newline

File: translations/en.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "app": {
     "name": "Linkets",
-    "short_description": "Link tinkets with Linkets!",
-    "long_description": "alexdglover is just one dude, trying to improve the world one app at a time.\nLinkets empowers your agents to link existing tickets to one another to show relationships, like when one ticket 'Blocks' another, or one ticket is a 'Parent' of another. This helps your agents find relevant tickets and information quickly! With visibility to related tickets and issues, your customer support agents will present a more unified, more cohesive experience to your customers.",
-    "installation_instructions": "Simply click *Install* button on the marketplace to install the app. Once the app is installed, Linkets will be displayed automatically on the right sidebar when viewing tickets. No configuration necessary!"
+    "short_description": "Widget with the ability to link tickets.",
+    "long_description": "Widget with the ability to link tickets.",
```
Comment (tlhinman) [inline translations/en.json:5]: Really expounding on it in that long description. :-D

**Review by tlhinman — APPROVED** [review]
I have deemed none of my comments blocking.

File: manifest.json (hunk 2 above)
Comment (Convale) [reply manifest.json:20]: Done

File: README.md (hunk 1 above)
Comment (Convale) [reply README.md:15]: Done

File: translations/en.json (hunk 3 above)
Comment (Convale) [reply translations/en.json:5]: Ha! I would have left it off if it wasn't a requirement. We're not posting to the Zendesk marketplace, so didn't feel the need to expand.

**Pull request merged**